hydra.gemspec

```diff
@@ -23,18 +23,16 @@ Gem::Specification.new do |gem|
   gem.require_paths = ["lib"]
   gem.license = 'APACHE2'
 
-  gem.add_dependency 'fcrepo_wrapper', '~> 0.5.1'
-  gem.add_dependency 'solr_wrapper', '~> 0.15.0'
-  gem.add_dependency 'active-fedora', '~> 10.0.0'
-  gem.add_dependency 'hydra-head', '~> 10.0.0'
-  gem.add_dependency 'rails', '~> 4.2'
-  gem.add_dependency 'om', '~> 3.1.1'
+  gem.add_dependency 'fcrepo_wrapper', '~> 0.7'
```

Comment:
Oh good point, I'll add that too. Dang, sorry I didn't see this before my last commit.